easybuild/easyconfigs/f/FigureGen/FigureGen-51-foss-2019a.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CmdCp'
+
+name = 'FigureGen'
+version = '51'
+
+homepage = 'https://ccht.ccee.ncsu.edu/figuregen-v-49/'
+description = """  FigureGen is a Fortran program that creates images for ADCIRC files.
+It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output 
+files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors. 
+Using FigureGen, you can go directly from the ADCIRC input and output files to a 
+presentation-quality figure, for one or multiple time snaps.  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'master.zip', 'filename': SOURCE_ZIP}]
+checksums = ['4199816638cf95ccd661eb0509c46d80c9dae986b2053a2db23cce293dcfa328']
+
+github_account = 'ccht-ncsu'
+files_to_copy = [
+    (['*'], '.'),
+]
```

The source code and temporary compilation files should not be copied to the installation directory. It is better to just copy the necessary files to use the software.
```suggestion
files_to_copy = [
    (['FigureGen.exe'], 'bin'),
    (['autotest/*'], 'autotest'),
    (['LICENSE'], ''),
]
```
Additionally, you might also consider moving `files_to_copy` after `cmds_map`, so that the parameters in the easyconfig follow the order of the build process.